src/lib/config.ts

```diff
@@ -1522,6 +1517,24 @@ export async function update() {
                 "vmaps",
             ].forEach(settingName => updateAll([settingName], updateSetting))
             set("configversion", "1.9")
+        }
+        case "1.9": {
```

Comment:
I've tested that this updater works migrating a toy profile.

I'm a bit nervous about merging the branch into master, though, as if I have got something wrong here I could mangle thousands of users configurations.

Could anyone look over it to see if there are any glaring errors? :)

Reply:
With my big old profile the migration doesn't work - I get left with a barebones local storage with configversion 1.10 and a plausible update.lastchecktime.

If I run
```
set configversion 1.9
jsb tri.config.update()
```
in a tab it works quite happily. I think our config updater probably doesn't work like we think it does.